bin/alluxio

```diff
@@ -11,7 +11,7 @@
 #
 
 function printUsage {
-  echo "Usage: alluxio COMMAND [GENERIC_COMMAND_OPTIONS] [COMMAND_ARGS]"
+  echo "Usage: alluxio [-debug] COMMAND [GENERIC_COMMAND_OPTIONS] [COMMAND_ARGS]"
```

Comment:
I think this is because the flag name is not better enough, so maybe the `-jvmdebug` is better, this flag is useful when you want to debug the shell command or any other command from `alluxio`, to let user know this flag well, i've write some description about this. In fact the flag is optional, other user can ignore this.